libc/src/stdio/fileno.h

```diff
@@ -0,0 +1,21 @@
+//===-- Implementation header of fileno --------------------------*- C++
+//-*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_SRC_STDIO_FILENO_H
+#define LLVM_LIBC_SRC_STDIO_FILENO_H
+
+#include <stdio.h>
```

Comment:
Please don't include <stdio.h> for hermeticity; that may be the system's headers rather than llvmlibc's.

You should probably be including "include/llvm-libc-types/FILE.h" instead.

Reply:
Updated.